deploy/vagrant/conf/tachyon_version.yml

```diff
@@ -8,3 +8,9 @@ Github:
 
 Release:
   Dist: tachyon-0.5.0-bin.tar.gz
+
+# how much memory to use (e.g. 1000MB, 2GB) per worker
+# make sure your virtual machine has more memory than what is specified here
+#  for virtualbox, machine memory is configured in conf/init.yml:Memory
+#  for aws, machine memory is decided by instance type
+Memory: 2GB
```

Comment:
This could be confusing, since there is another Memory: setting in another config file. It sounds like the other one is for the vm, and this is for the tachyon worker. Maybe call this "WorkerMemory" just to be distinct from the other setting?